README.md

```diff
@@ -57,25 +57,31 @@ This guide assumes:
 
 ## Configuration
 
-Configuration is started in GitHub and completed in Mattermost. 
+GitHub plugin configuration starts by registering an OAuth app in GitHub and ends in Mattermost. 
 
 **Note:** If you're using GitHub Enterprise, replace all GitHub links below with your GitHub Enterprise URL.
 
 ### Step 1: Register an OAuth Application in GitHub
 
+You must first register the Mattermost GitHub Plugin as an authorized OAuth app regardless of whether you're setting up the GitHub plugin as a system admin or a Mattermost user.
+
 1. Go to https://github.com/settings/applications/new to register an OAuth app.
 2. Set the following values:
-   - **Application Name:** `Mattermost GitHub Plugin - <your company name>`
+   - **Application name:** `Mattermost GitHub Plugin - <your company name>`
```

Comment:
I think we should use the same capitalization as GitHub here

Reply:
Agreed. Best practice is to match the case via the UI.